### PR TITLE
Add migration and availing services columns

### PR DESCRIPTION
**Pull request by bagnolil**
@emord @calellowitz 

Hey Cal and Emord, I'm not sure if the related doc expressions for the person case are accurate here. I wasn't able to find a straightforward example of a reference to the person case from another task case indicator so I used parent_id to start with. Could you check if that's the right way to reference the related person case from the task case? Thanks!

### CODE REVIEW

**Comment by emord** [comment]
@bagnolil I don't think that this will work. You should be able to use `icds_parent_parent_id` for this though. You can check its usage here: https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json#L2576-L2587

Also you should be able to test the syntax on ICDS by using https://www.icds-cas.gov.in/a/icds-cas/configurable_reports/expression_debugger/ you just need to specify a task case id for doc id and data source id can be `static-icds-cas-static-tasks_cases` since this is a task case

**Comment by bagnolil** [comment]
@emord I think the changes we discussed are implemented correctly. Let me know if there are any other issues with this. Thanks!

File: custom/icds_reports/ucr/data_sources/tasks_cases.json [hunk 1]
```diff
@@ -262,7 +262,76 @@
...
+			  "value_expression": {
+				"datatype": null, 
+				"type": "property_name", 
+				"property_name": "registered_status"
+			  }, 
+			  "type": "related_doc", 
+			  "related_doc_type": "CommCareCase", 
+			  "doc_id_expression": {
+				"type": "named", 
+				"name": "parent_id"
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/tasks_cases.json]: I think that registered_status is on the person case, correct? So this should use icds_parent_parent_id as well?

Comment (bagnolil) [reply custom/icds_reports/ucr/data_sources/tasks_cases.json]: Ah, right. Making that change now.

**Comment by bagnolil** [comment]
@emord made the change to availing services

**Pull request merged**